pkg/policy/opa/engine.go

```diff
@@ -419,8 +417,8 @@ func (e *Engine) Evaluate(engineInput policy.EngineInput) (policy.EngineOutput,
 	return e.results, nil
 }
 
-// FilterRules will apply the scan and skip rules, and severity level
-func (e *Engine) FilterRules(policyPath string, scanRules, skipRules []string, severity string) {
+// FilterRules will apply the scan and skip rules, severity level and categories
```

Comment:
```suggestion
// FilterRules will apply the scan and skip rules, categories and severity level
```

This is to maintain the same order as the function signature

Reply:
So, should I make change

**from this:**
// FilterRules will apply the scan and skip rules, severity level and categories
`func (e *Engine) FilterRules(policyPath string, scanRules, skipRules, categories []string, severity string)`

**to this:**
// FilterRules will apply the scan and skip rules, categories and severity level
`func (e *Engine) FilterRules(policyPath string, scanRules, skipRules [] string, severity string, categories []string)`

Please advise @dev-gaur